packages/images/test/unit/plugin-specs.js

```diff
@@ -118,7 +118,9 @@ describe('ImageElementPlugin#handle', function () {
       })};
       const el = await p.findElement(next, driver, IMAGE_STRATEGY, TEST_IMG_2_PART_B64);
       elId = el[W3C_ELEMENT_KEY];
-      await p.handle(next, driver, 'getAttribute', 'visual', elId).should.eventually.include('iVBOR');
+      // FIXME: temporary
```

Comment:
mm, need to take a look further..
```
@appium/images-plugin:       1) should return the match visualization as the visual attr
@appium/images-plugin:       ✓ should not allow any other attrs
@appium/images-plugin:   75 passing (1s)
@appium/images-plugin:   1 failing
@appium/images-plugin:   1) ImageElementPlugin#handle
@appium/images-plugin:        Element interactions
@appium/images-plugin:          should return the match visualization as the visual attr:
@appium/images-plugin:       AssertionError: expected '�PNG\r\n\u001a\n\u0000\u0000\u0000\rIHDR\u0000\u0000\u0000@\u0000\u0000\u0000@\b\u0002\u0000\u0000\u0000%\u000b�\u0000\u0000\u0002/IDATh\u0005��Aj\u0004A\f�@���V����CC���%�\u000b9�xI�B�*^R�����T.��%�\u000b9�xI�B�*^R�����T.��%�\u000b9�xI�B�*^R�����T.��%�\u000b9�xI�B�*^R�0�\u0017�1�\u001d90�\u001d90F��rQ��\f90F��rQ��\f90F��rQ��\f90F��rQ��\f90F��rQ��\f90F��rQ��\f90F��rQ��\f90F��rQ��\f90F��rQ��\f90F��rQ��\f90F��rQ��\f90F��rQ��\f90F��rQ��\f90F��rQ��\f90F��N|�5c\u0014o��ħX3F��N|�5c\u0014o��ħX3F��N|�5c\u0014o��ħX3F��N|�5c\u0014o��ħX3F��N|�5c\u0014o��ħX3F��N|�5c\u0014o��ħX3F��N|�5c\u0014o��ħX3F��N|�5c\u0014o��ħX3F��rQ���\u0014k�(vT.*���b�\u0018Ŏ�EŒ�S�\u0019��Q��XR~�5c\u0014;*\u0017\u0015K�O�f�bG�bI�)֌Q�\\T,)?Ś1�\u001d���%�X3F��rQ���\u0014k�(vT.*���b�\u0018Ŏ�EŒ�S�\u0019��Q��XR~�5c\u0014;*\u0017\u0015K�O�f�bG�bI�)֌��X3��b�\u000f\u0004�Y�����\u0000\u0000\u0000\u0000IEND�B`�' to deeply equal 'iVBOR'
@appium/images-plugin:       + expected - actual
@appium/images-plugin:       -�PNG
@appium/images-plugin:       -
@appium/images-plugin:       -
@appium/images-plugin: IHDR@@%
@appium/images-plugin: �/IDATh��AjA
@appium/images-plugin: �@���V����CC���%�
@appium/images-plugin: 9�xI�B�*^R�����T.��%�
@appium/images-plugin: 9�xI�B�*^R�����T.��%�
@appium/images-plugin: 9�xI�B�*^R�����T.��%�
@appium/images-plugin: 9�xI�B�*^R�0��1�90�90F��rQ��
@appium/images-plugin: 90F��rQ��
@appium/images-plugin: 90F��rQ��
@appium/images-plugin: 90F��rQ��
@appium/images-plugin: 90F��rQ��
@appium/images-plugin: 90F��rQ��
@appium/images-plugin: 90F��rQ��
@appium/images-plugin: 90F��rQ��
@appium/images-plugin: 90F��rQ��
@appium/images-plugin: 90F��rQ��
@appium/images-plugin: 90F��rQ��
@appium/images-plugin: 90F��rQ��
@appium/images-plugin: 90F��rQ��
@appium/images-plugin: 90F��rQ��
@appium/images-plugin: 90F��N|�5co��ħX3F��N|�5co��ħX3F��N|�5co��ħX3F��N|�5co��ħX3F��N|�5co��ħX3F��N|�5co��ħX3F��N|�5co��ħX3F��N|�5co��ħX3F��rQ���k�(vT.*���b�Ŏ�EŒ�S���Q��XR~�5c;*K�O�f�bG�bI�)֌Q�\T,)?Ś1����%�X3F��rQ���k�(vT.*���b�Ŏ�EŒ�S���Q��XR~�5c;*K�O�f�bG�bI�)֌��X3��b��Y�����IEND�B`�
@appium/images-plugin:       +iVBOR
@appium/images-plugin:       
@appium/images-plugin:       at Context.<anonymous> (test/unit/plugin-specs.js:123:32)
@appium/images-plugin:       at Generator.next (<anonymous>)
@appium/images-plugin:       at fulfilled (build/test/unit/plugin-specs.js:24:58)
@appium/images-plugin: npm ERR! code ELIFECYCLE
@appium/images-plugin: npm ERR! errno 1
@appium/images-plugin: npm ERR! @appium/images-plugin@1.1.5 test:unit: `mocha ./build/test/unit/*-specs.js`
@appium/images-plugin: npm ERR! Exit status 1
@appium/images-plugin: npm ERR! 
```

Reply:
looks like it's not being returned as b64

Reply:
yea, think so